server_ui/templates/confirm.html

```diff
@@ -38,12 +38,12 @@ <h3>Cast Your Ballot with your Credentials</h3>
 </form>
 
 <p>
-    Forgot your password? <a href="{% url "helios_auth.auth_systems.password.password_forgotten_view" %}?return_url={% url "server_ui.views.cast_confirm" %}">Have it emailed to you</a>.<br />(don't worry, we won't forget your vote).
```

Comment:
Have It emailed to you znamena nechat si ho poslat.
worry neni zoufat, ale mit obavu/strach.
vase bud velkym nebo tykat